test/core/crawlers/playwright_crawler.test.ts

```diff
@@ -153,4 +153,48 @@ describe('PlaywrightCrawler', () => {
 
         expect(Object.keys(options.browserPoolOptions).length).toBe(0);
     });
+
+    test.each([{ useIncognitoPages: true }, { useIncognitoPages: false }])(
+        'should apply launchOptions with useIncognitoPages: $useIncognitoPages',
+        async ({ useIncognitoPages }) => {
+            // Some launch options apply to the browser, while some apply to the context.
+            // Here we use some context options to verify that those are actually applied.
+            const launchOptions = {
```

Comment:
The original issue was about ignoring incorrect certificates, right? Could we also test that setting here?

Reply:
Could be an e2e test